data_enrichment.py

```diff
@@ -47,6 +48,35 @@ def format_nom_complet(
     return name.upper().strip() if name else name
 
 
+# Slug Nom complet
```

Comment:
Please add comment 
Also avoid using nom_complet. Maybe format_slug is the best as it is different from nom_complet